src/editableformfields/EditableDependentDynamicListField.php

```diff
@@ -4,7 +4,6 @@
 
 use SilverStripe\Forms\DropdownField;
 use SilverStripe\Forms\LiteralField;
```

Comment:
Pretty sure you can leave the `use` statement here and then do if(class_exists(EditableDropdown::class)) and things won't break. Is a little neater that way.

Reply:
This is exactly the comment I was about to make, beat me to it :)

Reply:
All done. Should be good to go now.